docs/src/starknet/declare-deploy.md

```diff
@@ -0,0 +1,32 @@
+# Joining the Declaration and Deployment Together
+
+`sncast` allows declaring and deploying contracts through dedicated commands.
+The `declare-deploy` command simplifies this pipeline by performing these two actions one after the other. It is used to declare a contract and deploy it immediately.
```

Comment:
```suggestion
The `declare-deploy` command simplifies this pipeline by performing these two actions at the same time. It is used to declare a contract and deploy it immediately.
```

Reply:
I know what you wrote is technically correct, but I think my suggestion is a better description for the user